### DISCUSSION TITLE
How can I autostart kdeconnect?

### DISCUSSION THREAD

**Discussion by ArjunSindhya** [i3/i3#4984] [Q&A]
I have i3-gaps on a minimal arcolinux install. I like kdeconnect, but I cannot get it to run at startup. 
I have added
`exec --no-startup-id kdeconnect-cli`
`exec --no-startup-id kdeconnect-indicator`
to my config but this doesn't work. Can anyone help?

**Comment by c02y** [comment]
I don't get kdeconnect intalled, but there is command `kdeconnect-app` in the kdeconnect package, you can try it.
